apps/myaccount/src/components/multi-factor-authentication/multi-factor-authentication.tsx

```diff
@@ -210,20 +212,38 @@ export const MultiFactorAuthentication: React.FunctionComponent<MfaProps> = (pro
                                     setShowSessionTerminationModal(true);
                                 } }
                             />
-                            { isSuperTenantLogin() && isTOTPEnabled && isBackupCodesConfigured
-                                ? (
-                                    <BackupCodeAuthenticator
-                                        onAlertFired={ onAlertFired }
-                                        initBackupCodeFlow={ initBackupCodeFlow }
-                                        onBackupFlowCompleted={ handleBackupCodeFlowCompleted }
-                                        handleSessionTerminationModalVisibility={
-                                            () => setShowSessionTerminationModal(true)
-                                        }
-                                    />
-                                ) : null
-                            }
                         </List.Item>
                     ) : null }
+                { hasRequiredScopes(featureConfig?.security, featureConfig?.security?.scopes?.read, allowedScopes) &&
```

Comment:
This feature should be able to be disabled by passing a key in to the `disabled_features` array of the feature config. Make sure to test that as well.

Ref: https://github.com/wso2/identity-apps/blob/9cac913a65c27205ed05807d05ba6e10047f4010/apps/myaccount/src/components/multi-factor-authentication/multi-factor-authentication.tsx#L160

Reply:
ack